### ISSUE TITLE
Load support file without SD card

### ISSUE THREAD

**Issue by Mich3le3D** [d0k3/GodMode9#270]
Support .bin files are stored in 0:/gm9/support.
I have installed GM9 in FIRM0 and putted aeskeydb.bin in SD card. If i boot the system without SD, i get error: "Not running from a boot9strap compatible entrypoint ... Provide the recomanded aeskeydb.bin ..."
In the next version of GM9 it's possible to able load support file from **1**:/rw/gm9/support, for example? Or it's already possible, but i do something wrong?

**Comment by figgyc** [comment]
You can precompile godmode9 with an aeskeydb built in but this is a good idea nonetheless.

**Comment by d0k3** [comment]
`1:/rw/gm9/support` does not work, you're forgetting about something important. On N3DS you need an external key to access even that. Thinking about it, though.

**Comment by ihaveamac** [comment]
yes, you can build GodMode9 with aeskeydb.bin in the data directory.

**Comment by d0k3** [comment]
Providing keys *is a grayzone*, same as you're not allowed to share the `boot9.bin` and `boot11.bin` you dumped from your console or any other Nintendo software. Thus, keys won't be hardcoded to GodMode9 - that decision was made long ago. I give you guys the choice to do so yourself, though, see above.

Now, I still get your problem. You want to use a release version of GM9, and run it, as bootloader, completely from NAND, right? I could maybe provide a way of installing an aeskeydb.bin to your NAND via the A button menu. Still thinking about it.

**Comment by Mich3le3D** [comment]
> run it, as bootloader, completely from NAND, right?
Right, but I'm not a developer, I'm not able to build custom version of GodMode9. Install an aeskeydb.bin on NAND is a good idea.

**Comment by Mich3le3D** [comment]
OK, thanks. The feature work well. Now GM9 don't need `aeskeydb.bin` in the SD card.
